BootstrapThreeThemePlugin.inc.php

```diff
@@ -135,6 +135,19 @@ public function init() {
 			$jquery = $request->getBaseUrl() . '/lib/pkp/lib/vendor/components/jquery/jquery' . $min . '.js';
 			$jqueryUI = $request->getBaseUrl() . '/lib/pkp/lib/vendor/components/jqueryui/jquery-ui' . $min . '.js';
 		}
+		
+		// Load icon font FontAwesome - http://fontawesome.io/
+		if (Config::getVar('general', 'enable_cdn')) {
+			$url = 'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css';
+		} else {
+			$url = $request->getBaseUrl() . '/lib/pkp/styles/fontawesome/fontawesome.css';
+		}
+		$this->addStyle(
+			'fontAwesome',
+			$url,
+			array('baseUrl' => '')
+		);
+		
```

Comment:
Bootstrap3 uses [glyphicons](https://getbootstrap.com/docs/3.3/components/) instead of FontAwesome. Rather than load another set of icons, we could:

a) Use one of the glyphicons

b) Don't use any icon. Now that you've separated the downloads from the other citation formats, I don't think we need an icon to distinguish between download and regular citations.

Reply:
No icons it is!